src/tests/ftest/server/metadata.py

```diff
@@ -397,12 +397,12 @@ def test_container_removal_after_der_nospace(self):
                         "##Expecting der_no_space RC: -1007, not seen")
                 else:
                     self.log.info(traceback.format_exc())
-                    self.log.info("(1.4)No space error shown with additional",
+                    self.log.info("(1.4)No space error shown with additional"
                                   "%d containers created, test passed.", cont)
                 break
         if cont == additional_container - 1:
             self.fail(
-                "##Storage resource did not exhaused after {} containers",
+                "##Storage resource did not exhausted after {} containers"
```

Comment:
I got this output:
TestFail: ##Storage resource did not exhausted after 4 containerscreated
You might want to add a space between "containers" and "created".